src/include/sof/samples/audio/kwd_nn_detect_test.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Cristina Feies <cristina.ilie@nxp.com>
+ * Author: Viorel Suman <viorel.suman@nxp.com>
+ */
+
+#ifndef __USER_KWD_NN_DETECT_TEST_H__
+#define __USER_KWD_NN_DETECT_TEST_H__
+
+#include <sof/samples/audio/kwd_nn/kwd_nn_config.h>
```

Comment:
Inconsistent include style between the two headers. One of them includes relative with "", the other includes absolute with <>. Either make them both use <> or make this one do "kwd_nn/kwd_nn_config.h".